[2021-07-16 16:46:40,220] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:46:07.631606+00:00 [queued]>
[2021-07-16 16:46:40,259] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:46:07.631606+00:00 [queued]>
[2021-07-16 16:46:40,260] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:46:40,261] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:46:40,262] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:46:40,279] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:46:07.631606+00:00
[2021-07-16 16:46:40,295] {standard_task_runner.py:52} INFO - Started process 9352 to run task
[2021-07-16 16:46:40,302] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:46:07.631606+00:00', '--job-id', '1240', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpczynazq_', '--error-file', '/tmp/tmpb4mlzkal']
[2021-07-16 16:46:40,303] {standard_task_runner.py:77} INFO - Job 1240: Subtask restart_test
[2021-07-16 16:46:40,384] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:46:07.631606+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:46:40,475] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:46:07.631606+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:46:04.902434+00:00
[2021-07-16 16:46:44,172] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T164607, start_date=20210716T164640, end_date=20210716T164644
[2021-07-16 16:46:44,238] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:46:44,297] {local_task_job.py:151} INFO - Task exited with return code 0
